assemble/conf/templates/accumulo-env.sh

```diff
@@ -33,24 +48,22 @@ JAVA_OPTS=("${ACCUMULO_JAVA_OPTS[@]}" '-XX:+UseConcMarkSweepGC' '-XX:CMSInitiati
 ## JVM options set for individual applications
 case "$ACCUMULO_CMD" in
 master)  JAVA_OPTS=("${JAVA_OPTS[@]}" ${masterHigh_masterLow}) ;;
+monitor) JAVA_OPTS=("${JAVA_OPTS[@]}" ${monitorHigh_monitorLow}) ;;
 gc)      JAVA_OPTS=("${JAVA_OPTS[@]}" ${gcHigh_gcLow}) ;;
 tserver) JAVA_OPTS=("${JAVA_OPTS[@]}" ${tServerHigh_tServerLow}) ;;
-monitor) JAVA_OPTS=("${JAVA_OPTS[@]}" ${monitorHigh_monitorLow}) ;;
 shell)   JAVA_OPTS=("${JAVA_OPTS[@]}" ${shellHigh_shellLow}) ;;
 *)       JAVA_OPTS=("${JAVA_OPTS[@]}" ${otherHigh_otherLow}) ;;
 esac
-export JAVA_OPTS
 
-## Accumulo logs directory. Referenced by logger config.
-export ACCUMULO_LOG_DIR="${ACCUMULO_LOG_DIR:-$ACCUMULO_HOME/logs}"
-## Hadoop installation
-export HADOOP_PREFIX="${HADOOP_PREFIX:-/path/to/hadoop}"
-## Hadoop configuration
-export HADOOP_CONF_DIR="${HADOOP_CONF_DIR:-${HADOOP_PREFIX}/etc/hadoop}"
-## Zookeeper installation
-export ZOOKEEPER_HOME="${ZOOKEEPER_HOME:-/path/to/zookeeper}"
-## See HADOOP-7154 and ACCUMULO-847
-export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
+## JVM options set for logging
+JAVA_OPTS=("${JAVA_OPTS[@]}" "-Daccumulo.application=${ACCUMULO_CMD}" "-Daccumulo.log.dir=${ACCUMULO_LOG_DIR}" "-Daccumulo.local.hostname=$(hostname -s)")
```

Comment:
The `man` page for `hostname` actually recommends avoiding the use of `-f`, because it may have multiple, or zero fully-qualified domain names, which is unreliable for how we're using it here. We probably just want `hostname`, without `-s`, because that's literally the local host name, unshortened. It'd be up to name service configuration to make sure this is consistent with the name on the network.